package-lock.json

```diff
@@ -287,7 +241,6 @@
       }
     },
     "libs/ansible": {
-      "name": "@flightctl/ansible",
```

Comment:
why was this removed ? We still have libs/ansible and it has name `@flightctl/ansible`

Reply:
It's how it got recreated, I also though it was weird.
I can revert manually, but we may need to take a look.